lib/inquirer.js

```diff
@@ -47,6 +47,7 @@ inquirer.createPromptModule = function (opt) {
     // promise object so they remain publicly accessible.
     promise.rl = ui.rl;
     promise.process = ui.process;
+    promise.ui = ui;
```

Comment:
Would it be too cumbersome to remove `promise.rl` and reference it as `promise.ui.rl` in the tests?